pkg/web/templates/accounts.html

```diff
@@ -95,6 +95,6 @@ <h1 class="font-bold text-xl">Add New Customer Account</h1>
 
 {{ define "appcode" }}
 <script src="https://unpkg.com/slim-select@latest/dist/slimselect.min.js"></script>
-<script src="/static/js/customerAccounts.js"></script>
-<script src="/static/js/networkSelect.js"></script>
+<script type="module" src="/static/js/customerAccounts.js"></script>
+<script type="module" src="/static/js/networkSelect.js"></script>
```

Comment:
I've never seen this before - very cool. I'm glad module functionality and import/export is coming to the browser, even if it is just limited availability right now.